scripts/build_docker_image.sh

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+# Directory above this script
+SUBNET_EVM_PATH=$( cd "$( dirname "${BASH_SOURCE[0]}" )"; cd .. && pwd )
+
+# Load the constants
+source "$SUBNET_EVM_PATH"/scripts/constants.sh
+
+# Load the versions
+source "$SUBNET_EVM_PATH"/scripts/versions.sh
+
+# WARNING: this will use the most recent commit even if there are un-committed changes present
+BUILD_IMAGE_ID=${BUILD_IMAGE_ID:-"${CURRENT_BRANCH}"}
```

Comment:
removed `avalanchego-{version}` from image ID as we won't have any commit with different avalanchego versions. (having a different avalanchego version mean a different commit)